.github/actions/deploy-macosx/action.yml

```diff
@@ -108,6 +108,11 @@ runs:
         export DYLD_LIBRARY_PATH="$LIBRARY_PATH"
         export DYLD_FALLBACK_LIBRARY_PATH="$LIBRARY_PATH"
 
+        # Change default JVM on macos-14 runner
+        if [[ -n ${JAVA_HOME_17_arm64:-} ]]; then
+          export JAVA_HOME=$JAVA_HOME_17_arm64
+        fi
+
```

Comment:
Java 8 or 11 isn't available??

Reply:
> Java 8 or 11 isn't available??

In the github runner, 11 is, 8 is not.
We can install 8 from brew.
But I don't see the interest.

Reply:
I don't think Java 8 is available for arm64. Meh, Java 17 is fine I guess.